bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx

```diff
@@ -19,6 +20,8 @@ const VideoListItem = (props) => {
     cameraId, numOfStreams, focused, onVideoItemMount, onVideoItemUnmount,
   } = props;
 
+  const mirrorOwnWebcam = VideoService.mirrorOwnWebcam(user.userId);
+
   const [videoIsReady, setVideoIsReady] = useState(false);
   const [isStreamHealthy, setIsStreamHealthy] = useState(false);
   const [isMirrored, setIsMirrored] = useState(false);
```

Comment:
I think the PR could be made simpler/clearer, but correct me if I'm missing some edge case.

If the initial state of `isMirrored` was the return of `VideoService.mirrorOwnWebcam(user.userId)`, you could remove the added `mirrorOwnWebcam` style prop added in L151.  You could then remove the updates to styles.js, essentially turning this PR into a 2 liner.

```
diff --git a/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx b/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx
index 64d955630b..b3be4dac74 100755
--- a/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx
+++ b/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx
@@ -20,11 +20,9 @@ const VideoListItem = (props) => {
     cameraId, numOfStreams, focused, onVideoItemMount, onVideoItemUnmount,
   } = props;
 
-  const mirrorOwnWebcam = VideoService.mirrorOwnWebcam(user.userId);
-
   const [videoIsReady, setVideoIsReady] = useState(false);
   const [isStreamHealthy, setIsStreamHealthy] = useState(false);
-  const [isMirrored, setIsMirrored] = useState(false);
+  const [isMirrored, setIsMirrored] = useState(VideoService.mirrorOwnWebcam(user.userId));
 
   const videoTag = useRef();
   const videoContainer = useRef();
@@ -148,7 +146,6 @@ const VideoListItem = (props) => {
         <Styled.Video
           muted
           mirrored={isMirrored}
-          mirrorOwnWebcam={mirrorOwnWebcam}
           unhealthyStream={shouldRenderReconnect}
           data-test={isMirrored ? 'mirroredVideoContainer' : 'videoContainer'}
           ref={videoTag}
diff --git a/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/styles.js b/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/styles.js
index fbf518bb5a..c390363965 100644
--- a/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/styles.js
+++ b/bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/styles.js
@@ -141,7 +141,7 @@ const Video = styled.video`
   object-fit: contain;
   background-color: ${colorBlack};
 
-  ${({ mirrorOwnWebcam, mirrored }) => ((!mirrorOwnWebcam && mirrored) || (mirrorOwnWebcam && !mirrored)) && `
+  ${({ mirrored }) => mirrored && `
     transform: scale(-1, 1);
   `}
```

Reply:
Yeah, it's simpler this way.